MessengerOnFileForLinux/UserService/test/SignOutTestCore.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include "SignOut.hpp"
+#include "FileHandling.hpp"
+#include "GlobalVariables.hpp"
+
+class SignOutTestFixture : public ::testing::Test
+{
+public:
+    SignOutTestFixture() {}
+    void SetUp() {}
+    void TearDown() {}
+    ~SignOutTestFixture() {}
+
+    void signInUser()
+    {
+        const std::string USER = getenv("USER");
+        const std::string rowInLoggedFile = "[" + USER + "][0]";
```

Comment:
Mamy do tego klasę  StringSum

Reply:
Remove User class and getting 100% coverage of LocalUser